src/date/parse.js

```diff
@@ -1,13 +1,11 @@
 define([
-	"./is-leap-year",
-	"./last-day-of-month",
+	"../core",
 	"./pattern-re",
 	"./start-of",
 	"../common/create-error/unsupported-feature",
-	"../util/date/set-month",
 	"../util/out-of-range"
-], function( dateIsLeapYear, dateLastDayOfMonth, datePatternRe, dateStartOf,
-	createErrorUnsupportedFeature, dateSetMonth, outOfRange ) {
+], function( Globalize, datePatternRe, dateStartOf,
```

Comment:
We shouldn't need `Globalize` in here. Having calendars object in a different place (as commented above) will solve this.